lib/lunchbox_bot.ex

```diff
@@ -4,12 +4,14 @@ defmodule LunchboxBot do
   @base_endpoint "https://slack.com/api/"
   @channel_info_path "/channels.info"
   @post_message_path "/chat.postMessage"
+  @user_info_path "/users.info"
   @default_greeting """
   Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
   Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
   Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
   Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
   """
+  @ignore_people ["hugh", "ricardo", "victor"]
```

Comment:
does Slack give access to each users notification preferences for a channel? maybe ignore people who have notifications turned off